ssas/service/admin/router_test.go

```diff
@@ -0,0 +1,30 @@
+package admin
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+type RouterTestSuite struct {
+	suite.Suite
+	router  http.Handler
+}
+
+func (s *RouterTestSuite) SetupTest() {
+	s.router = NewRouter()
+}
+
+func (s *RouterTestSuite) TestPostSystemRoute() {
+	req := httptest.NewRequest("POST", "/system", nil)
+	rr := httptest.NewRecorder()
+	s.router.ServeHTTP(rr, req)
+	res := rr.Result()
+	assert.Equal(s.T(), http.StatusBadRequest, res.StatusCode)
+}
+
+func TestRouterTestSuite(t *testing.T) {
```

Comment:
Only a negative test? Why no positive test?

Reply:
This is testing the path only, because it's for the router. There are other tests for the handler function.